Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM alpine:3.14 as default
```

Comment:
Has someone more closely involved in docker releasing (e.g. @jasonodonnell) reviewed this already?  Are we planning on getting rid of the docker-vault repo?